app/src/main/java/com/openclassrooms/realestatemanager/ui/LoanSimulatorActivityViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.openclassrooms.realestatemanager.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import kotlin.math.pow
+
+class LoanSimulatorActivityViewModel: ViewModel() {
+
+    private var monthlyPaymentLiveData: MutableLiveData<Double> = MutableLiveData(0.0)
+    private var totalInterestsLiveData: MutableLiveData<Double> = MutableLiveData(0.0)
+    private var totalPayment: MutableLiveData<Double> = MutableLiveData(0.0)
+
+    fun getMonthlyPaymentLiveData() = monthlyPaymentLiveData
+
+    fun getTotalInterestsLiveData() = totalInterestsLiveData
+
+    fun getTotalPaymentLiveData() = totalPayment
```

Comment:
En Kotlin, je retrouve souvent les variables mutables privées en avec le suffix `_` et les valeurs publiques non mutables définit de la sorte (

```suggestion
    private val _monthlyPaymentLiveData: MutableLiveData<Double> = MutableLiveData(0.0)
    val monthlyPaymentLiveData 
        get() = _monthlyPaymentLiveData as LiveData<Double>
    
    // or
    
    private val _totalInterestsLiveData: MutableLiveData<Double> = MutableLiveData(0.0)
    val _totalInterestsLiveData: LiveData<Double> 
        get() = _totalInterestsLiveData
    ```

Reply:
un exemple : https://developer.android.com/codelabs/basic-android-kotlin-training-livedata?hl=fr#3

Reply:
Ok très interessant merci

Reply:
Je vais resté sur mon écriture actuel car cela correspond aux cours Openclassrooms, et j'ai peut de tomber sur un évaluateur qui ne connaît pas.

Reply:
https://github.com/Tgarfr/Project-9-RealEstateManager/pull/9/commits/b63ae5962956a79244292f97b6587b51d02d9a6e